career/general_personality.md

```diff
@@ -0,0 +1,93 @@
+
+### Explain how your favorite application works in a way a layman would understand.
+
+People skills
+ Programmers generally aren’t expected to have heavy-duty customer service talents. It seems to be understood at the outset that they’ll mostly be sitting in front of a computer all day. Yet programmers are regularly expected to interact with managers, co-workers, and clients, so the ability to work well with others is a must. This is especially true if your programmers are occasionally asked to participate in client meetings and explain how a system works.
+
+### Tell us about a time when you saved time by automating a process.
+
+Laziness (?!?)
+
+Larry Wall, the original author of, The Perl Programming Languages, describes three good qualities in a good programmer: laziness, impatience and hubris. Laziness may sound like a bad trait for any employee to have, but IT managers have said that if you want to find the best way to do something, ask a lazy person. Chances are, that person will have found the quickest, most efficient way to do it. A company’s programming staff can often find a way to automate processes, saving time and money.
+
+### Have you ever made a suggestion that improved a business’s processes?
+
+Business perspective
+
+It’s easy to lose sight of the big picture while focusing on creating one piece of software. The ideal programmer has a business focus that allows him to move beyond the current application. A business-focused programmer will suggest ideas for new applications that can improve operations.
+
+### Explain your approach to a new design. What do you do first?
+
+Ability to plan
+
+Rather than jumping into a new assignment, a good addition to your programming staff will first learn as much as possible about the desired end product. Once he’s completed that analysis, the programmer will first design the program structure before typing the first line of code.
+
+
+### Have you ever spent hours on a code, only to find it was the wrong approach? What did you do?
+
+Ability to handle failure
+
+Programmers rarely get everything right on the first try. In fact, failure is almost a certainty. It’s important to find programming staff that see errors and bugs as a challenge rather than a sign of defeat. Persistence is important, as well as the ability to start over if necessary, even after hours of work.
+
+### Can you tell us about  a rewarding team experience?
+
+Teamwork mentality
+
+A programmer rarely works alone, even if he’s the only developer in a company. The ability to work with business users, marketing and sales staff, and fellow programmers is essential.
```

Comment:
`The ability to work with business users, marketing and sales staff, and fellow programmers is essential.`
`The ability to work with business users, marketing and sales staff and fellow programmers is essential.`